packages/tracing/src/browser/browsertracing.ts

```diff
@@ -190,7 +190,7 @@ export class BrowserTracing implements Integration {
 
     // eslint-disable-next-line @typescript-eslint/unbound-method
     const { beforeNavigate, idleTimeout, maxTransactionDuration } = this.options;
-
+    const hub = this._getCurrentHub();
```

Comment:
You got me curious, why move the declaration up?!

Reply:
Leftover from PoC. Restored.